msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/Constants.java

```diff
@@ -23,4 +26,13 @@ final class Constants {
     public static final String MSI_ENDPOINT = "MSI_ENDPOINT";
     public static final String IDENTITY_SERVER_THUMBPRINT = "IDENTITY_SERVER_THUMBPRINT";
 
+    // Constants for token revocation and client capabilities
+    public static final String TOKEN_HASH_CLAIM = "token_sha256_to_refresh";
+    public static final String CLIENT_CAPABILITY_REQUEST_PARAM = "xms_cc";
+    
+    // Only Service Fabric and App Service managed identity environments support token revocation
+    public static final Set<ManagedIdentitySourceType> TOKEN_REVOCATION_SUPPORTED_ENVIRONMENTS = new HashSet<ManagedIdentitySourceType>() {{
+        add(ManagedIdentitySourceType.SERVICE_FABRIC);
```

Comment:
The comment says that SF and app service support revocation, so should ManagedIdentitySourceType.APP_SERVICE be added here too?

Reply:
I am updating the comment for now to only have service fabric here